class.jetpack-options.php

```diff
@@ -552,6 +552,7 @@ static function get_all_wp_options() {
 			'jpo_site_type',
 			'jpo_homepage_format',
 			'jpo_contact_page',
+			'jetpack_extensions_exclusions',
```

Comment:
Nit: I'd likely go with `jetpack_excluded_extensions` as this would make the option name more consistent with our naming scheme, which often is `(jetpack_)?(ADJECTIVE_)?(NOUN)`.

Definitely not a requirement and no hard feelings about this one (I assume the option name is already decided), just wanted to share my $0.02. 😉

Reply:
> I assume the option name is already decided

That sounds reasonable to me as well. Should be a simple fix.

Reply:
I've updated this everywhere to be jetpack_excluded_extensions 🙂